canonical/vanilla-framework
# Adds class for anchor links to have visual onhover state

## Done

Adds class for anchor links to have visual onhover state

Fixes [list issues/bugs if needed]

## QA

- Open [demo](https://vanilla-framework-5120.demos.haus/docs/examples/patterns/links/links-anchor?theme=light)
- Make sure that the anchor icon appears on hover and focus as expected
- Review updated documentation:
  - https://vanilla-framework-5120.demos.haus/docs/patterns/links#anchor-link

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical/vanilla-framework/wiki/Release-process#pre-release-tasks)


## Screenshots

![anchors](https://github.com/canonical/vanilla-framework/assets/83575/8e729bdf-3446-4535-8a27-c1dc2c73d4d1)
